Tools/GNUMake/Make.machines

```diff
@@ -127,3 +127,8 @@ ifneq (,$(filter $(host_name_env),poplar redwood tulip))
   which_computer := $(host_name_env)
 endif
 
+ifeq ($(findstring birch, $(host_name)),birch)
+  which_site := frontier-coe
+  which_computer := birch
+endif
+
```

Comment:
Should other frontier-coe sites also check for `cray.com`, and / or be combined into one if test?
```suggestion
ifeq ($(findstring "cray.com", $(host_name)),"cray.com")
ifeq ($(findstring birch, $(host_name)),birch)
  which_site := frontier-coe
  which_computer := $(host_name_env)
endif
endif

```

Reply:
I cannot remember how the host name for other coe machines looked like before the recent changes to the systems.

Reply:
If `hostname -f` all contains `cray.com` now, we can put them together.

Reply:
when I check `hostname -f` it looks like that won't work for tulip. Will just adding an extra check for birch be sufficient for now?

Reply:
Yes.